CHANGELOG.md

```diff
@@ -1,3 +1,146 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* Create and set-up data partition at runtime ([#2807], [#2879], [#2873])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Add `CAP_SYS_MODULE` and `CAP_CHROOT` to bootstrap containers ([#2772])
+* Add support for cgroup v2 ([#2875], [#2802])
+* Disable IA and SafeSetID LSM for kernel-5.15 ([#2789])
+* Update kernel-5.10 to 5.10.165 and kernel-5.15 to 5.15.90 ([#2795])
+* Allow `=` in bootconfig values ([#2806])
+* Include `systemd-analyze plot` for `logdog` ([#2880])
+* Update host containers ([#2864])
+* Upgrade `go-restful` code dependency used by containerd to avert unaffected CVE ([#2813])
+* Update third party packages ([#2825], [#2842])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Add Kubernetes 1.26 variants ([#2771], ([#2876])
+* Change systemd `kubelet` service to have restart policy `always` ([#2774])
+* Update `eni-max-pod` mappings to include the latest AWS instance types ([#2818])
+* Remove Kubernetes 1.21 variants ([#2700])
+* Update to Kubernetes v1.25.6 ([#2782])
+* Update to Kubernetes v1.24.10 ([#2790])
+* Update to Kubernetes v1.23.16 ([#2791])
+* Update Kubernetes 1.22.17 to include latest EKS-D patches ([#2792])
+
+## Platform Changes
+
+### VMware
+
+* Removes `k8s.gcr.io` in favor of `public.ecr.aws` ([#2861], ([#2786])
```

Comment:
Nit: verb tense should align for all these changelog entries:
``suggestion
* Remove `k8s.gcr.io` in favor of `public.ecr.aws` ([#2861], ([#2786])
```